src/TableActionCell/TableActionCell.js

```diff
@@ -55,14 +56,23 @@ function renderHiddenActions(actions, popoverMenuProps) {
     <PopoverMenu
       buttonTheme="icon-greybackground"
       dataHook="table-action-cell-popover-menu"
-      appendToParent
+      appendTo="parent"
+      placement="top"
```

Comment:
Let's add the following to be aligned with the previous design:
```suggestion
      placement="top"
      textSize="small"
```